interp/api.go

```diff
@@ -457,7 +457,7 @@ func stdinFile(r io.Reader) (*os.File, error) {
 // Note that providing a non-nil standard input other than [os.File] will require
 // an [os.Pipe] and spawning a goroutine to copy into it,
 // as an [os.File] is the only way to share a reader with subprocesses.
-// See [os/exec.Cmd.Stdin].
+// See os/[exec.Cmd.Stdin].
```

Comment:
I'm undoing this change as it seems unnecessary and makes the linking less consistent.